codersdk/deployment.go

```diff
@@ -682,7 +686,7 @@ type NotificationsConfig struct {
 
 type NotificationsEmailConfig struct {
 	// The sender's address.
-	From serpent.String `json:"from" typescript:",notnull"`
+	From runtimeconfig.Entry[*serpent.String] `json:"from" typescript:",notnull"`
```

Comment:
All of these email & webhook configs will be runtime-configurable; this is just the starting point.